OpenDreamClient/Interface/DreamInterfaceManager.cs

```diff
@@ -498,34 +505,92 @@ internal sealed class DreamInterfaceManager : IDreamInterfaceManager {
             return true;
         }
 
+        string HandleEmbeddedWinget(string? controlId, string value) {
+            string result = value;
+            int startPos = result.IndexOf("[[");
```

Comment:
IndexOf is culture sensitive. Probably not a big deal with `[[` and `]]` but still good practice to specify how to compare the strings.
```suggestion
            int startPos = result.IndexOf("[[", StringComparison.Ordinal);
```